web-components/src/components/organisms/Gallery/Gallery.tsx

```diff
@@ -56,9 +56,6 @@ const Gallery = ({ photos, sx }: Props) => {
           },
         }}
       >
-        {/* // TODO Update framer-motion to latest version and remove comments below after NextJS upgrade */}
```

Comment:
no need to wait for the nextjs upgrade since in the mean time we migrated to vite, so we don't depend on react-scripts anymore